src/System.Runtime.Serialization.Xml/pkg/ValidationSuppression.txt

```diff
@@ -1,2 +1,3 @@
 // Remove once we can OOB this to desktop
 PermitPortableVersionMismatch=.NETFramework,Version=v4.6.1;.NETFramework,Version=v4.6
+PermitHigherCompatibleImplementationVersion=.NETCore,Version=v5.0;.NETStandard,Version=v1.6;.NETCoreApp,Version=v1.0
```

Comment:
Thanks for the explanation. Maybe worth adding a comment that will be helpful for future understanding.